src/D2L.Security.OAuth2/Keys/Default/TokenSigner.cs

```diff
@@ -36,6 +37,8 @@ IPrivateKeyProvider privateKeyProvider
 					signingCredentials: securityToken.GetSigningCredentials()
 				);
 
+				jwt.Payload.Add(OAuth2.Constants.Claims.ISSUED_AT, ((DateTimeOffset)token.IssuedAt).ToUnixTimeSeconds());

```

Comment:
Would prefer to use the implicit constructor vs an explicit cast. Explicit casts can hide conversions that are never valid.

Also nit on whitespace
```suggestion
				DateTimeOffset issuedAtOffset = token.IssuedAt;
				jwt.Payload.Add( OAuth2.Constants.Claims.ISSUED_AT, issuedAtOffset.ToUnixTimeSeconds() );
```